lms/djangoapps/teams/tests/test_api.py

```diff
@@ -144,6 +144,30 @@ def test_get_team_invalid_course(self):
         with self.assertRaisesMessage(ValueError, message):
             teams_api.get_team_for_user_course_topic(self.user1, invalid_course_id, 'who-cares')
 
+    def test_get_team_anonymous_user_ids(self):
```

Comment:
These are good tests.  How about one more to cover the condition where either the user or the team passed to `get_team_anonymous_user_ids` is falsey?